biz.aQute.junit/src/aQute/junit/plugin/ProjectTesterImpl.java

```diff
@@ -29,6 +30,10 @@ public ProjectTesterImpl(Project project) throws Exception {
 		super(project);
 	}
 
+	static String maybeQuote(String s) {
+		return (s.indexOf(',') == -1) ? s : '"' + s + '"';
```

Comment:
It is a bit more complicated than this. The user may have already quoted the string! `--tests="class:method(int,int)"`.

See https://github.com/bndtools/bnd/blob/593fc85bbb09dd6b4c1e65ae4637142987e2019a/aQute.libg/src/aQute/libg/qtokens/QuotedTokenizer.java#L222-L250 for a robust quoting example.

Reply:
Wouldn't it be better if the process responsible for populating testernames ensure that the strings that go into the list are normalized to begin with?

Reply:
This means that every driver must do this instead of in fewer places. Better to do this at the choke points.

Reply:
Better to do this at the choke points, yes, but I think that the choke point is where the model is populated (`ProjectTester.addTest()`). That way, the drivers don't have to worry about it either (as they all use `addTest()`). If I do it in `ProjectTesterImpl` I have to do it three times, plus any other third-party consumers of `getTests()` will also have to handle it.

Reply:
I have added code to strip quotes (if present) in `ProjectTester.addTest()`.